web/src/styles/questions/question-one-styles.css

```diff
@@ -0,0 +1,73 @@
+.sequential-intro-1 {
+    position:fixed;
+    z-index: 1;
+    animation: disappear 0s forwards;
+    animation-delay: 7s;
+}
+
+@keyframes disappear {
+    to {
+        visibility: hidden;
+        z-index: -1;
+	}
+}
+/*------------------------*/
+
+.sequential-child:nth-child(1):after {
+    width: 0px;
+    overflow: hidden;
+    display: inline-block;
+    vertical-align: bottom;
+    -webkit-animation: ellipsis steps(4,end) 4s forwards;     
+    animation: ellipsis steps(4,end) 4s forwards;
+    animation-iteration-count: 1;
+    content: "\2026"; /* ascii code for the ellipsis character */
+  }
+  
+  @keyframes ellipsis {
+    to {
+      width: 1.8em;    
+    }
+  }
+  
+  @-webkit-keyframes ellipsis {
+    to {
+        width: 1.8em;    
+    }
+  }
+
+/*------------------------*/
+
+.sequential-child:nth-child(2) {
+    opacity: 0;
+    animation: appear 100ms forwards;
+    animation-delay: 4s;
+}
+
+@keyframes appear {
+    to {
+		opacity: 1;
+	}
+}
+
+/*------------------------*/
+
+.sequential-intro-2 {
+    position:fixed;
+    visibility: hidden;
+
+    animation: intro-2-appear 100ms forwards;
+    animation-delay: 7s;
+}
+
+@keyframes intro-2-appear {
+    to {
+        visibility: visible;
+	}
+}
+
+.qintro__cta {
+    opacity: 0;
+    animation: appear 100ms forwards;
+    animation-delay: 9s;
```

Comment:
I think this can be cut to like 3 or 4 seconds. 9 feels a bit too long for me, like something didn't load properly on the site long